src/core_plugins/kibana/public/management/sections/indices/add_data_steps/parse_csv_step/parse_csv_step.js

```diff
@@ -72,6 +72,10 @@ modules.get('apps/management')
                     this.formattedErrors.push('Column names must be unique');
                   }
 
+                  if (results.meta.fields.some(name => /\s/.test(name))) {
+                    this.formattedWarnings.push('Field names with spaces are dificult to search for in Kibana');
```

Comment:
How the hell could that happen, I copied your sentence from IRC :D

Anyway... fixed it.